Modules/Office365DSC/DSCResources/MSFT_SPOTenantSettings/readme.md

```diff
@@ -0,0 +1,5 @@
+
+# SPO Tenant Settings
+
+This resource allows users to configure and monitor the tenant settings for
+your SPO tenant settings.
```

Comment:
can we change to "... tenant settings for their SPO tenant"?

their instead of your

Reply:
sure :-). Done